superflore/__init__.py

```diff
@@ -1,11 +1,6 @@
-try:
-    import pkg_resources
-    try:
-        __version__ = pkg_resources.require('superflore')[0].version
-    except pkg_resources.DistributionNotFound:
-        __version__ = 'unset'
-except (ImportError, OSError):
-    __version__ = 'unset'
+from superflore.version import VERSION
```

Comment:
This convoluted indirection was an attempt to work around local import trouble with the RepoInstance. It may be we don't need the superflore.version module if the tests are better behaved in general than they are in my local workspace.